sample-plugin/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>bootstrap4-api</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
is this needed here? I thought api plugins weren't supposed to be listed here?

Reply:
The build failure said it is needed in the sample plugin:
> Managed plugin dependency io.jenkins.plugins:caffeine-api not listed in test classpath of sample plugin

https://ci.jenkins.io/blue/organizations/jenkins/Tools%2Fbom/detail/PR-505/1/pipeline#step-20-log-322

Reply:
That is why this would have been better paired with #504 as a single PR: you would I think get `caffeine-api` as a transitive dep in the sample plugin.

Reply:
⇒ #507